Description: Make SSL_OP_MSIE_SSLV2_RSA_PADDING optional
 Since 0.9.7h SSL_OP_MSIE_SSLV2_RSA_PADDING has no effect and
 it have been removed in version 0.9.8a
 .
 This patch makes it optional.
Author: Mehdi Abaakouk <sileht@sileht.net>
Origin: https://bugs.launchpad.net/pyopenssl/+bug/1266521
Bug: https://bugs.launchpad.net/pyopenssl/+bug/1266521
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=733366
Forwarded: not-needed
Last-Update: 2014-01-12
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/OpenSSL/ssl/ssl.c
+++ b/OpenSSL/ssl/ssl.c
@@ -211,7 +211,9 @@ do {
     PyModule_AddIntConstant(module, "OP_NETSCAPE_REUSE_CIPHER_CHANGE_BUG", SSL_OP_NETSCAPE_REUSE_CIPHER_CHANGE_BUG);
     PyModule_AddIntConstant(module, "OP_SSLREF2_REUSE_CERT_TYPE_BUG", SSL_OP_SSLREF2_REUSE_CERT_TYPE_BUG);
     PyModule_AddIntConstant(module, "OP_MICROSOFT_BIG_SSLV3_BUFFER", SSL_OP_MICROSOFT_BIG_SSLV3_BUFFER);
+#ifdef SSL_OP_MSIE_SSLV2_RSA_PADDING
     PyModule_AddIntConstant(module, "OP_MSIE_SSLV2_RSA_PADDING", SSL_OP_MSIE_SSLV2_RSA_PADDING);
+#endif
     PyModule_AddIntConstant(module, "OP_SSLEAY_080_CLIENT_DH_BUG", SSL_OP_SSLEAY_080_CLIENT_DH_BUG);
     PyModule_AddIntConstant(module, "OP_TLS_D5_BUG", SSL_OP_TLS_D5_BUG);
     PyModule_AddIntConstant(module, "OP_TLS_BLOCK_PADDING_BUG", SSL_OP_TLS_BLOCK_PADDING_BUG);
